### DISCUSSION TITLE
Internal Calculation error

### DISCUSSION THREAD

**Discussion by coca7** [mr-manuel/venus-os_dbus-serialbattery#49] [General]
I keep getting an Internal calculation error.  I thought this might be because I started using Dr. Gigavolts Battery aggregator, but there are no errors in Battery Aggregators logs.

In my VRM I get the following Error:
Battery Monitor - SerialBattery(LLT/JBD) [1]	Error: Internal calculation error

When I checked the logs:

> 2024-07-09 00:28:20.948612500 ERROR:SerialBattery:>>> ERROR: Invalid response packet. Expected begin packet character 0xDD
> 2024-07-09 00:28:20.950426500 WARNING:SerialBattery:>>> WARN: BMS rejected request. Status 29
> 2024-07-09 00:48:37.618993500 ERROR:SerialBattery:>>> ERROR: Invalid response packet. Expected begin packet character 0xDD
> 2024-07-09 00:48:37.620169500 ERROR:SerialBattery:>>> ERROR: BMS send insufficient data. Received 16 expected 7
> 

There is one other piece of information.  I had to RMA my Cerbo GX and the batteries got badly out of balance without the Cerbo and Serial Battery.  The battery which gives the error most often is the most out of balance.  Once cell is high.

**Comment by mr-manuel** [comment]
Please open a support issue, so I get all data needed.
